lib/measured/rails/active_record.rb

```diff
@@ -14,12 +15,20 @@ def measured(measured_class, *fields)
 
       fields.map(&:to_sym).each do |field|
         raise Measured::Rails::Error, "The field #{ field } has already been measured" if measured_fields.keys.include?(field)
+
         measured_fields[field] = options
 
+        if options[:unit_field_name]
+          unit_field_name = options[:unit_field_name]
+          measured_fields[field][:unit_field_name] = unit_field_name
```

Comment:
You need to be opinionated about string vs symbol. Because since you don't coerce here your `include?` below can incorrectly report `"length_unit"` and `:length_unit` as different.